<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>591</width>
      <height>281</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>11</y>
       <width>256</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>46</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;BFS遍历结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>381</x>
       <y>46</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;队列变化情况&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>221</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>186</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>151</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>116</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>81</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>46</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>31</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>278</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>93</y>
       <width>256</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>31</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;显示邻接链表&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>591</width>
      <height>260</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>11</y>
       <width>256</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>46</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>81</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>11</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>116</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>151</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>186</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>221</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>46</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;DFS遍历结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>46</y>
       <width>83</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;栈变化情况&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>31</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>31</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>240</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>textBrowser_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>line_3</zorder>
    <zorder>line_4</zorder>
    <zorder>line_5</zorder>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>16</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>275</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>889</x>
      <y>300</y>
      <width>31</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>新建(&amp;F)</string>
    </property>
    <addaction name="actioncreate"/>
   </widget>
   <widget class="QMenu" name="menuDFS">
    <property name="title">
     <string>遍历(&amp;E)</string>
    </property>
    <addaction name="actiondfsnostack"/>
    <addaction name="separator"/>
    <addaction name="actiondfsstack"/>
    <addaction name="separator"/>
    <addaction name="actionbfs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuBFS">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
    <addaction name="actionaboutqt"/>
    <addaction name="separator"/>
    <addaction name="actionaboutauther"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDFS"/>
   <addaction name="menuBFS"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actioncreate"/>
   <addaction name="separator"/>
   <addaction name="actiondfsnostack"/>
   <addaction name="separator"/>
   <addaction name="actiondfsstack"/>
   <addaction name="separator"/>
   <addaction name="actionbfs"/>
   <addaction name="separator"/>
   <addaction name="actionhelp"/>
   <addaction name="separator"/>
   <addaction name="actionaboutqt"/>
   <addaction name="separator"/>
   <addaction name="actionaboutauther"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiondfsstack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/dfs_stack.png</normaloff>:/icon/myicon/dfs_stack.png</iconset>
   </property>
   <property name="text">
    <string>非递归DFS</string>
   </property>
   <property name="toolTip">
    <string>非递归方式深度优先遍历</string>
   </property>
  </action>
  <action name="actionbfs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/bfs.png</normaloff>:/icon/myicon/bfs.png</iconset>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
   <property name="toolTip">
    <string>广度优先遍历</string>
   </property>
  </action>
  <action name="actioncreate">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/create.png</normaloff>:/icon/myicon/create.png</iconset>
   </property>
   <property name="text">
    <string>建图</string>
   </property>
   <property name="toolTip">
    <string>邻接矩阵方式确定一个图</string>
   </property>
  </action>
  <action name="actiondfsnostack">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/dfs_nostack.png</normaloff>:/icon/myicon/dfs_nostack.png</iconset>
   </property>
   <property name="text">
    <string>递归DFS</string>
   </property>
   <property name="toolTip">
    <string>递归方式深度优先遍历</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/help.png</normaloff>:/icon/myicon/help.png</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
   <property name="toolTip">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="actionaboutqt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/qt.png</normaloff>:/icon/myicon/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
   <property name="toolTip">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionaboutauther">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/myicon/aboutauther.png</normaloff>:/icon/myicon/aboutauther.png</iconset>
   </property>
   <property name="text">
    <string>关于作者</string>
   </property>
   <property name="toolTip">
    <string>关于作者</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
